jdaviz/configs/imviz/plugins/catalogs/catalogs.py

```diff
@@ -11,11 +12,19 @@
                                         FileImportSelectPluginComponent, HasFileImportSelect,
                                         with_spinner)
 
+from jdaviz.core.template_mixin import TableMixin
+from jdaviz.core.user_api import PluginUserApi
+
+
+
 __all__ = ['Catalogs']
 
 
 @tray_registry('imviz-catalogs', label="Catalog Search")
-class Catalogs(PluginTemplateMixin, ViewerSelectMixin, HasFileImportSelect):
+
+#Katherine added in TableMixin 
```

Comment:
No need to add this as a comment, we can see who added what using gits "blame" feature - which is not as a aggressive as it sounds!